java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/map/prunability/KeyPojo.java

```diff
@@ -0,0 +1,38 @@
+package com.hazelcast.simulator.tests.map.prunability;
+
+import com.hazelcast.partition.PartitionAware;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class KeyPojo implements Serializable, PartitionAware<Integer> {
```

Comment:
could you explain why `KeyPojo` is `PartitionAware`? This is very tricky configuration because `AttributePartitioningStrategy` ignores `PartitionAware`. It this correct at all?

Reply:
yes, I need it for PredicateAPI usage. I will show it in the next commit.

Reply:
add a comment explaining that. This is not obvious and IMO a bad practice for the users (you define you object as `PartitionAware` but it ignored by the strategy) but clever hack for the tests